x/blob/ante/blob_share_decorator_test.go

```diff
@@ -67,7 +66,20 @@ func TestMaxTotalBlobSizeAnteHandler(t *testing.T) {
 			// single blob that is 2 MiB returns an error.
 			wantErr: true,
 		},
-		// tests based on shares
+		{
+			name: "PFB with many single byte blobs should fit",
+			pfb: &blob.MsgPayForBlobs{
+				BlobSizes: repeat(4095, 1), // (64 * 64) - 1 = 4095 so this should fit
```

Comment:
[Question] What is the size of each share in the square? it seems to be assumed to be 1 byte?

Reply:
this is trying to convey that if there are 4095 blobs each of one byte then the entire PFB should fit. Recall that even if a blob only contains one byte, it must occupy an entire blob share because multiple blobs cannot occupy one share.

So this could theoretically be `repeat(4095, 100)` or 4095 blobs each of size 100 bytes because a single 100 byte blob occupies a single share.

Reply:
Clarified in 9c448ae940b500fae327095b5f3cd049adca1b02